src/pages/Experiments/Experiment/Drawer/ResultsDrawer/ResultsDrawerSkeleton.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Skeleton, Row, Col, Card } from 'antd';
+
+const ResultsDrawerSkeleton = () => {
+  const skeletonColumn = (
+    <Col span={12}>
+      <Card
+        className='monitoring-drawer-skeleton-card'
```

Comment:
Rename these class names starting with "monitoring-drawer" to "results-drawer" or remove them if they are not used anymore.